v7.13.0 (#14221)
mui/mui-x: CHANGELOG.md

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 7.13.0

_Aug 15, 2024_

We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:

- 💫 Allow label editing capabilities on the Tree View (#13388) @noraleonte

> - 🔧 Improve rows accessibility on the Data Grid Tree Data and Row Grouping features (#13623) @arminmeh

Reviewer: ```suggestion
- 🔧 Improve rows accessibility for Data Grid features "Tree Data" and "Row Grouping". See the [documentation](https://mui.com/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) for more information. (#13623) @arminmeh
```

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## 7.13.0

> _Aug 15, 2024_

Reviewer 2: ```suggestion
_Aug 16, 2024_
```

We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:

> - 💫 Allow [label editing capabilities on the Tree View](https://mui.com/x/react-tree-view/rich-tree-view/editing/) (#13388) @noraleonte

Reviewer 3: ```suggestion
- 💫 Allow to [edit the label](https://mui.com/x/react-tree-view/rich-tree-view/editing/) of Tree View's items.
```

From what I can see we don't put the author and the PR link on highlights
If we want to put them it would be nice to discuss and align :+1: 

And "capabilities" seems slightly convoluted here.

Reviewer 3: @noraleonte this on may be worth an inline video

> - 💫 Allow [label editing capabilities on the Tree View](https://mui.com/x/react-tree-view/rich-tree-view/editing/) (#13388) @noraleonte

Reviewer 4: ```suggestion
- 💫 Allow [label editing capabilities on the Tree View](https://mui.com/x/react-tree-view/rich-tree-view/editing/) (#13388) @noraleonte
  <img width="344" src="https://github.com/user-attachments/assets/1a6cf765-2dc8-4906-bd93-139086eed148" alt="Item label editing" />
```

Author: nice! I added it ;)

> - 🔧 Improve rows accessibility on the Data Grid features "Tree Data" and "Row Grouping". See the [documentation](https://mui.com/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) for more information. (#13623) @arminmeh

Reviewer 3: ```suggestion
- 🔧 Improve rows accessibility on the Data Grid features "Tree Data" and "Row Grouping". See the [documentation](https://mui.com/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) for more information.
```

Reviewer 5: ```suggestion
- 🔧 Improve rows accessibility on the Data Grid features "Tree Data" and "Row Grouping". Certain "Row Grouping" accessibility updates will only be applied if experimental feature flag is enabled. See the [documentation](https://mui.com/x/react-data-grid/row-grouping/#accessibility-changes-in-v8) for more information.
```

Author: I took @arminmeh suggestion because it adds an additional explanation ;)

- [DataGrid] Fix error logged during skeleton loading with nested data grid (#14186) @KenanYusuf
- [DataGrid] Remove needless check in `useGridStateInitialization` (#14181) @k-rajat19
- [l10n] Improve Vietnamese (vi-VN) locale (#14216) @hungnd-casso

#### `@mui/x-data-grid-pro@7.13.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-data-grid@7.13.0`, plus:

- [DataGridPro] Fix Tree Data and Row Grouping rows accessibility (#13623) @arminmeh

#### `@mui/x-data-grid-premium@7.13.0` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')

Same changes as in `@mui/x-data-grid-pro@7.13.0`.

### Date and Time Pickers

#### `@mui/x-date-pickers@7.13.0`

- [pickers] Fix date and time merging to retain milliseconds (#14173) @LukasTy

#### `@mui/x-date-pickers-pro@7.13.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')

Same changes as in `@mui/x-date-pickers@7.13.0`.

### Charts

#### `@mui/x-charts@7.13.0`

- [charts] Add `baseline` property to the `LineChart` `series` (#14153) @JCQuintas

> - [charts] Add real-world dataset (#13958) @alexfauquette

Reviewer 6: ```suggestion
```

> - [charts] Do not remove mouse position for mouseUp event (#14187) @alexfauquette

Reviewer 6: ```suggestion
- [charts] Fix issue where tooltip would disappear on mouse click (#14187) @alexfauquette
```

- [charts] Rename `CartesianContextProvider` to `CartesianProvider` (#14102) @JCQuintas

> - [charts] Support axis with single value (#14191) @alexfauquette

Reviewer 6: ```suggestion
- [charts] Support axis with the same value for all data points (#14191) @alexfauquette
```
